airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -2801,9 +2801,7 @@ def _split_tablename(
 ) -> Tuple[str, str, str]:
 
     if '.' not in table_input:
-        raise ValueError(
-            'Expected target table name in the format of ' '<dataset>.<table>. Got: {}'.format(table_input)
-        )
+        raise ValueError(f'Expected table name in the format of <dataset>.<table>. Got: {table_input}')
```

Comment:
Fix the earlier static check error.

Please note that I made a minor change to the content here for two reasons:
- otherwise the updated line will be longer than 110
- more importantly, this `_split_tablename` method is used for both `source` and `destination`. Word `target` may be a bit misleading, and the updated one is meaning enough to me as well.

CC @mik-laj